File: crates/npt/src/color.rs
//! Thin styling helpers over `console`, which handles tty detection, `NO_COLOR`,
//! and enabling ANSI on Windows automatically.

use console::style;

pub fn ok(s: &str) -> String {
    style(s).green().to_string()
}

pub fn warn(s: &str) -> String {
    style(s).yellow().to_string()
}

pub fn err(s: &str) -> String {
    style(s).red().to_string()
}

/// Bold cyan — used for titles/headers.
pub fn title(s: &str) -> String {
    style(s).cyan().bold().to_string()
}

/// Cyan — used for values worth highlighting (package name, binding).
pub fn accent(s: &str) -> String {
    style(s).cyan().to_string()
}

pub fn dim(s: &str) -> String {
    style(s).dim().to_string()
}

File: src/color.ts
//! Terminal styling. picocolors handles tty detection and `NO_COLOR`.

import pc from 'picocolors'

export const ok = (s: string): string => pc.green(s)
export const warn = (s: string): string => pc.yellow(s)
export const err = (s: string): string => pc.red(s)
export const title = (s: string): string => pc.bold(pc.cyan(s))
export const accent = (s: string): string => pc.cyan(s)
export const dim = (s: string): string => pc.dim(s)
